### PR TITLE
feat: eXOF helm charts, celotool changes and other adhoc fixes

### PR DESCRIPTION
**Pull request by nvtaveras**
### Description

This branch contains helm charts for the eXOF oracles which will be soon deployed as well and some other changes/fixes that piled up little by little that we didn't manage to merge before. 

### Other changes

- Use light nodes by default on celotool deployments 
- Some fixes related to the API-keys secret
- USDC/USD helm charts updates 
- Other changes to the charts that were done in the past month or so due so config issues on the clients

### Tested

This is the branch from which all our mainnet and testnets oracles are running from.

### Related issues

- Fixes https://github.com/mento-protocol/mento-general/issues/245

### Backwards compatibility

- No, since it not deploys clients using light nodes instead of the previous default (full nodes)

### Documentation

-

### CODE REVIEW

File: .env.alfajores [hunk 1]
```diff
@@ -23,20 +23,23 @@ CELOSTATS_BANNED_ADDRESSES=""
 CELOSTATS_RESERVED_ADDRESSES=""
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="us-west1-docker.pkg.dev/celo-testnet-production/celo-oracle/celo-oracle"
-ORACLE_DOCKER_IMAGE_TAG="1.0.3"
+ORACLE_DOCKER_IMAGE_TAG="latest"
```
Comment (bowd) [inline .env.alfajores:26]: Do we want to keep this on latest?

File: .env.baklava [hunk 2]
```diff
@@ -36,7 +36,7 @@ CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
 CELOCLI_STANDALONE_IMAGE_TAG="0.0.30-beta2"
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="us-west1-docker.pkg.dev/celo-testnet-production/celo-oracle/celo-oracle"
-ORACLE_DOCKER_IMAGE_TAG="1.0.3"
+ORACLE_DOCKER_IMAGE_TAG="latest"
```
Comment (bowd) [inline .env.baklava:39]: ditto

**Review by bowd — APPROVED** [review]
👍

File: .env.alfajores (hunk 1 above)
Comment (nvtaveras) [reply .env.alfajores:26]: Yeah will do, as that's what's currently deployed on testnets. Will update it soon anyways once we do a new release

**Pull request merged**